instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/db/DbClientSpanNameExtractor.java

```diff
@@ -25,11 +25,11 @@ public static <REQUEST> SpanNameExtractor<REQUEST> create(
 
   /**
    * Returns a {@link SpanNameExtractor} that constructs the span name according to DB semantic
-   * conventions: {@code <db.operation> <db.name>.<table>}.
+   * conventions: {@code <db.operation> <db.name>.<identifier>}.
    *
    * @see SqlStatementInfo#getOperation() used to extract {@code <db.operation>}.
    * @see DbClientAttributesGetter#name(Object) used to extract {@code <db.name>}.
-   * @see SqlStatementInfo#getTable() used to extract {@code <db.table>}.
+   * @see SqlStatementInfo#getIdentifier() used to extract {@code <db.identifier>}.
```

Comment:
not exactly sure how to phrase it, but `db.identifier` isn't a semantic attribute
```suggestion
   * @see SqlStatementInfo#getIdentifier() used to extract {@code <db.table>} or stored procedure name.
```